runtime/mount/agent.py

```diff
@@ -699,6 +659,57 @@ async def submit_response(args: dict) -> str:
                 })
         self.tool_map["set_widget"] = set_widget
 
+    async def _get_notebook_context(self) -> str:
```

Comment:
You might need to change some things in the prompt that reference calling get_notebook_context
e.g. 
`* If a step in the plan is related to a widget, call get_notebook_context to get its current`